app/Livewire/Notifications/Discord.php

```
@@ -163,6 +163,30 @@ public function instantSaveDiscordEnabled()
+            $this->resetErrorBag();
+
+            if ($this->discordEnabled) {
+                $this->discordEnabled = false;
+            } else {
+                $this->validate([
+                    'discordWebhookUrl' => 'required',
+                ], [
+                    'discordWebhookUrl.required' => 'Discord Webhook URL is required.',
+                ]);
+                $this->discordEnabled = true;
+            }
```

PHPMD (2.15.0)

[error] The method toggleDiscordEnabled uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)